### DISCUSSION TITLE
sudo -u postgres psql :  Permission denied

### DISCUSSION THREAD

**Discussion by elmundo78** [netbox-community/netbox#15397] [Q&A]
Good morning ,
I follow the documentation on the link https://docs.netbox.dev/en/stable/installation/1-postgresql/ but I have problems when I type the command: sudo -u postgres psql, I receive this message: root@netbox:/home/user# sudo -u postgres psql
could not change directory to "/home/user": Permission denied
psql (14.11 (Ubuntu 14.11-0ubuntu0.22.04.1))
Type "help" for help.
 someone to help me please?

**Comment by candlerb** [comment]
There's no real problem there: you got the psql prompt, and you can run psql commands just fine.

The message is only saying that you're currently in /home/user but the postgres user isn't allowed to be there. You can do `cd /tmp` first if you want to avoid that warning.

**Reply by elmundo78** [reply]
thank you for your help, in tmp I no longer have the error
